saleor/graphql/product/mutations/products.py

```diff
@@ -540,6 +540,9 @@ class ProductInput(graphene.InputObjectType):
             "Note: this field is only used if a product doesn't use variants."
         )
     )
+    visible_in_listings = graphene.Boolean(
+        description="Determines if product is visible in product listings."
```

Comment:
Don't we want to have default value here?

Reply:
The default value is in the model field definition. I think it's enough. :)